management-model/src/main/java/org/terracotta/management/call/ContextualReturn.java

```diff
@@ -30,16 +34,20 @@
  *
  * @author Mathieu Carbou
  */
-public final class ContextualReturn<T> implements Serializable {
+public final class ContextualReturn<T> implements Externalizable {
 
-  private final T value;
-  private final Context context;
-  private final String capability;
+  private T value;
+  private Context context;
+  private String capability;
+
+  @Deprecated
```

Comment:
The no-arg ctor is required for deserialization. And @Deprecated is used to discourage its use and warn the user.

Management object model wasn't designed for java serialization in mind...

`@Deprecated` javadoc: 

"A program element annotated @Deprecated is one that programmers ___are discouraged from using___, typically because it is ___dangerous___, or because a ___better alternative___ exists. Compilers warn when a deprecated program element is used or overridden in non-deprecated code."